### DISCUSSION TITLE
I have question regarding tags in tailwind css

### DISCUSSION THREAD

**Discussion by 2div** [tailwindlabs/tailwindcss#11419] [General]
Dear community,

 In my component i have more than h1 tags and more than image , when for example click on the hamburger icon i want the below image move down so that nav items displayed , i gave sm: mt-20 to image but this effect other image margins , how can i only target that image only ?

**Comment by wongjn** [comment]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) or a remotely-hosted git repository that reproduces your problem – it would be difficult to offer any solutions from the description you gave without any concrete code to look at.

**Comment by 2div** [comment]
I am new to Tailwind , i will try to provide repo but i would like to give you a simple example:

<header>
    <h1 className='mt-5'>
             Hello world
   </h1>
</header>
<section>
   <h1 className='mt-6'>
           Hello world 2
   </h1>
</section>  */

when i try to style the h1 in the section its effect the h1 in header , same issue happens with Image tag in NextJS

In i normal css , i can target h1 in header by using ' header h1' , how to do it in tailwind css ?

Thanks

**Reply by wongjn** [reply]
You'd want to look at adding classes as you'd like to the `<h1>` in the `<header>` like: https://play.tailwindcss.com/l1oNPvXpSM

**Reply by 2div** [reply]
Yes i can add classes in normal css but I am using NextJS with Tailwind

**Reply by wongjn** [reply]
You can still add classes in NextJS. NextJS uses React, so you can follow [the React syntax for adding classes](https://react.dev/reference/react-dom/components/common#applying-css-styles):

> In React, you specify a CSS class with [className.](https://developer.mozilla.org/en-US/docs/Web/API/Element/className) It works like the class attribute in HTML:
>
> ```jsx
> <img className="avatar" />
> ```

So with `<h1>` elements, you'd do something like:

```jsx
<h1 className="text-red-500">Foo</h1>
```

**Reply by 2div** [reply]
Ok , Thank you for explaining
